sympy/ntheory/tests/test_ntheory.py

```diff
@@ -381,12 +381,17 @@ def test_issue_6981():
 
 
 def test_totient():
+    from sympy.core import symbols
     assert [totient(k) for k in range(1, 12)] == \
         [1, 1, 2, 2, 4, 2, 6, 4, 6, 4, 10]
     assert totient(5005) == 2880
     assert totient(5006) == 2502
     assert totient(5009) == 5008
     assert totient(2**100) == 2**99
+    x = symbols('x')
+    assert totient(x) == totient(x)
```

Comment:
There are some other test cases which needs to be added too.
Kindly look at this issue. https://github.com/sympy/sympy/issues/8922 .
Once I get a heads up on this, I can add the test case related to it.
Thanks.